docs/README.md

```diff
@@ -126,14 +126,14 @@ You can exclude a heading from the table of contents:
 
 	{% top %}
 
-This will be replaced by a default back to top link. It is recommended to use these links at least at the end of each level-2 section.
+This will be replaced by default back to top link. It is recommended to use these links at least at the end of each level-2 section.
```

Comment:
This sounds a bit wrong; what's the reasoning for changing it?

Reply:
The link mentioned here is not specific. Hence I think the article used can be removed when the context is more generic.